packages/docs/content/components/footer/healthcare-footer.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: Healthcare.gov Footer
```

Comment:
I'm not sure why, but for some reason the sidebar is capitalizing `.Gov` 🤷‍♂️. Not a big deal, but pointing it out

Reply:
Oh... I think the sidebar has a css text transform on it to capitalize text and that was leftover from when we were using filenames not frontmatter titles. Let me change that and see if we can remove the styling